pages/settings/wallets/[wallet].js

```diff
@@ -44,7 +44,7 @@ export default function WalletSettings () {
       <Form
         initial={initial}
         schema={schema}
-        onSubmit={async (values) => {
+        onSubmit={async ({ amount, ...values }) => {
```

Comment:
I'm not sure if we need `amount` anymore but this was getting saved in the configs for wallets

Reply:
Ah no, thanks for fixing it. I also noticed that but always when I was already doing something else and then I forgot.